ast/Cargo.toml

```diff
@@ -16,6 +16,8 @@ fold = []
 unparse = ["rustpython-literal"]
 visitor = []
 all-nodes-with-ranges = []
+malachite-bigint = ["dep:malachite-bigint"]
+num-bigint = ["dep:num-bigint"]
```

Comment:
Do we need these lines? Isn't an optional dependency name itself a dependency name? Please let me know if I am missing recently changed recommendation or guideline.

Reply:
Yes, you are right. these are not needed. I will update this and the PR to RustPython as well.